### DISCUSSION TITLE
After the upgrade, consumption is missing in VRM historical data

### DISCUSSION THREAD

**Discussion by johnyhravy** [Louisvdw/dbus-serialbattery#260] [Q&A] [answered]
Hello.
Yesterday I upgraded from v0.12 to v0.13 and the consumption stopped showing in the VRM dashboard historical data.
Is this happening to any of you or is it just me?
Thanks

**Comment by johnyhravy** [comment]
Hello.
So I think I found the answer. I have JK BMS and SmartShunt and if JK BMS is selected in the automatic monitoring, the consumption is not displayed, and today the automatic machine selected SmartShunt and the consumption is already shown.
I could not ask to modify the program so that SmartShunt is hard set for the purpose of consumption display.
Thank you.

**Comment by Louisvdw** [comment]
You can change the Battery Monitor that is selected in your GX settings. Just change it from Auto to which device you want to use.

**Comment by johnyhravy** [comment]
Thanks for the reply, but I've read a lot of posts where you mostly advise if we have a BMS to choose it because it will give the system more information to charge the battery. Or did I get it wrong?
Thank you.

**Reply by Louisvdw** [reply, marked as answer]
Using the BMS does give you more data to work with, so I always suggest you use the BMS as the BMS. 
But if you want to use the consumption from other device then you can choose it as the Battery Monitor. But what you select there is what the GX system will use. It' nog easy to merge data from the serial battery with other devices.

**Reply by MAY4VFX** [reply]
It seems I'm facing the same problem, I have the latest stable build at the moment, and I have daily bms, but if daily bms is selected in the battery monitor settings and not smart shunt, then I don't see consumption, does this mean that there is some kind of limitation in my bms or is it a plugin problem?
